src/markspelling.py

```diff
@@ -47,17 +49,22 @@ def checkline(self, line, linenumber, filename, incodeblock=False):
         incodeblock = self.checkcodeblock(line, incodeblock)
         errorwords = list()
         if not wasincodeblock and not incodeblock:
-            self.logger.debug('Checking line "%s"', line.rstrip())
+            self.logger.debug('%03d   RAW %s', linenumber, line.rstrip())
```

Comment:
Why a triple space?

Reply:
Formatting, debugging is easier when the output is lined up.

Reply:
Ah, right.